src/pipelines/vaccinations/us_authority_vaccination_type.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+from functools import partial
+from lib.concurrent import process_map
+from typing import Dict
+from pandas import DataFrame, concat, read_csv
+from lib.cached_data_source import CachedDataSource
+from lib.utils import table_rename
+from lib.io import read_file
+
+
+_column_adapter = {
+    'Series_Complete_Yes': 'total_persons_fully_vaccinated',
+    'Doses_Administered': 'total_vaccine_doses_administered',
+    'Administered_Dose1_Recip':'total_persons_vaccinated',
+
+    'Series_Complete_Pfizer': 'total_persons_fully_vaccinated_pfizer',
+    'Administered_Pfizer': 'total_vaccine_doses_administered_pfizer',
+
+    'Series_Complete_Moderna': 'total_persons_fully_vaccinated_moderna',
+    'Administered_Moderna': 'total_vaccine_doses_administered_moderna',
+
+    'Series_Complete_Janssen': 'total_persons_fully_vaccinated_janssen',
+    'Administered_Janssen': 'total_vaccine_doses_administered_janssen',
+
+    "key": "key"
+    }
+
+us_states = ['US'] + read_csv("data/us_cdc_locations.csv").key.values.tolist()
```

Comment:
Sorry I just noticed this, one last request:

```suggestion
us_states = ['US'] + read_csv(SRC / "data" / "us_cdc_locations.csv").key.values.tolist()
```

You can import `SRC` from `lib.constants`. This will make sure that running the code works no matter what your current directory is.